### PR TITLE
inclusive terminology for Greenplum 7 in FDW

### PR DESCRIPTION
**Pull request by hyongtao-db**
rename master to coordinator under these 4 directories：
- gpcontrib/gp_exttable_fdw 
- contrib/postgres_fdw
- contrib/file_fdw
- src/backend/access/external

**Signed-off-by**: Yongtao Huang <yongtaoh@vmware.com>

### CODE REVIEW

File: src/include/nodes/plannodes.h [hunk 1]
```diff
@@ -912,15 +912,15 @@ typedef struct WorkTableScan
...
-	bool		isMasterOnly;   /* true for EXECUTE on master seg only */
-	int			rejLimit;       /* reject limit (-1 for no sreh)      */
-	bool		rejLimitInRows; /* true if ROWS false if PERCENT      */
-	char		logErrors;      /* 't', 'p' to log errors into file. 'p' makes persistent error log */
-	int			encoding;		/* encoding of external table data    */
-	uint32      scancounter;	/* counter incr per scan node created */
-	List	   *extOptions;		/* external options */
+	List		*uriList;            /* data uri or null for each segment  */
+	char		fmtType;             /* data format type                   */
+	bool		isCoordinatorOnly;   /* true for EXECUTE on master seg only */
```
Comment (huiliang-liu) [inline src/include/nodes/plannodes.h:917]: on `coordinator` seg only

File: src/backend/access/external/external.c [hunk 2]
```diff
@@ -954,7 +954,7 @@ create_external_scan_uri_list(ExtTableEntry *ext, bool *ismasteronly)
 			 * meant for the master segment (not seg o).
```
Comment (huiliang-liu) [inline src/backend/access/external/external.c:954]: coordinator

Comment (hyongtao-db) [reply src/backend/access/external/external.c]: done

File: src/include/nodes/plannodes.h (hunk 1 above)
Comment (hyongtao-db) [reply src/include/nodes/plannodes.h]: done

**Review by huiliang-liu — APPROVED** [review]

**Pull request merged**